registry.json

```diff
@@ -102,5 +102,17 @@
   "telegram": "HYPERLINKPOOL",
   "operatorName": "",
   "github": "hyperlinkpool"
- }
-]
+},
+{
+ "ticker": "THOR",                                                      
+ "poolId": "40d04c1367c25e6da2521f906f71bbb07858dc1d2209025364369173",
+ "social": {                                                          
+   "twitter": "Vikings_for_ADA",
+   "facebook": "THOR - Cardano ADA Staking Pool",  
+   "instagram": " ",
+   "youtube": " "
+ },
+ "telegram": "THOR The Viking Pool",                                   
```

Comment:
please create a Telegram handle and use that

Example:
![image](https://user-images.githubusercontent.com/4371293/94070773-7896f880-fdfb-11ea-9fe7-fc57c2f3a201.png)